--- ./.venv3.6/lib/python3.6/site-packages/netzob/Import/PCAPImporter/PCAPImporter.py	2025-02-04 23:17:08.046758942 +0900
+++ ./.venv3.6/lib/python3.6/site-packages/netzob/Import/PCAPImporter/PCAPImporter.py	2025-02-04 23:17:37.767808767 +0900
@@ -40,8 +40,8 @@
 # import impacket.ImpactDecoder as Decoders
 # import impacket.ImpactPacket as Packets
 ## Instead, import local adapted files
-from netzob.Import.PCAPImporter import ImpactPacket as Packets
-from netzob.Import.PCAPImporter import ImpactDecoder as Decoders
+from impacket import ImpactPacket as Packets
+from impacket import ImpactDecoder as Decoders
 
 #+---------------------------------------------------------------------------+
 #| Local application imports
